src/pushsource/_impl/schema/errata-schema.yaml

```diff
@@ -149,6 +155,8 @@ properties:
         $ref: "#/definitions/reference_list"
     pkglist:
         $ref: "#/definitions/pkg_collection_list"
+    containerlist:
```

Comment:
For the schema to work correctly, the property name has to match the name you're looking for in `_from_data` method.  Right now the schema is going to check a property named `containerlist` but then the code ends up using a field named `container_list`, so it's not actually being checked by the schema.

So either this should be `container_list` or from_data should be looking for `containerlist`.

Reply:
fixed